packages/components/atoms/f-form-field/src/components/_tests/FormField.test.js

```diff
@@ -241,4 +241,41 @@ describe('FormField', () => {
             });
         });
     });
+
+    describe('events ::', () => {
+        describe('@input ::', () => {
+            it.each(VALID_INPUT_TYPES)('should only trigger one event emission for inputType `%s`', inputType => {
+                // Arrange
+                const propsData = { inputType };
+                const wrapper = shallowMount(FormField, { propsData });
+
+                const formInput = wrapper.find('input');
+
+                // Act
+                formInput.trigger('input');
```

Comment:
Just a thought, but would it be better to actually change the value of the input and check that the input event was triggered?